app/assets/stylesheets/components/_inset_text.scss

```diff
@@ -0,0 +1,3 @@
+.govuk-inset-text {
+  white-space: pre-line;
+}
```

Comment:
This feels like a bit of a code smell to me 😕

Presumably you've done this because you can't put `<br>` tags into the translations you've defined?

https://github.com/DFE-Digital/itt-mentor-services/blob/0f0bd1b4a1b734c527b205e4494c4cbabd9b8100/config/locales/en/claims/schools/claims/mentor_trainings.yml#L11-L19

I'd rather we didn't define CSS rules to work around this limitation. This approach breaks developer's expectations of how HTML and CSS typically render, and it won't apply to any other element or component. So if we ever needed to change the inset to something else – for example, if a future design called for us to use a [notification banner](https://govuk-components.netlify.app/components/notification-banner/) instead – then we'd need to apply `white-space: pre-line` to the notification banner CSS instead. This doesn't feel like an obvious or sensible coupling to me – it feels a bit hacky.

I also note that there's some repetition in the translations themselves – they're not very 'DRY'. It suggests to me that they're probably two different translated strings.

What about splitting the lines up into separate translations? I think this would be cleaner and allow you to use the appropriate HTML in the inset element.